src/main/java/org/jenkinsci/test/acceptance/po/Job.java

```diff
@@ -188,10 +209,18 @@ public void copyFile(File file) {
             ZipUtil.pack(file, tmp);
             byte[] archive = IOUtils.toByteArray(new FileInputStream(tmp));
 
-            addShellStep(String.format(
-                    "base64 --decode << ENDOFFILE > archive.zip && unzip -o archive.zip \n%s\nENDOFFILE",
-                    new String(Base64.encodeBase64Chunked(archive))
-            ));
+            if (SystemUtils.IS_OS_WINDOWS) {
+                if (!(controller instanceof LocalController)) {
+                    // TODO: Make it work for RemoteJenkinsController like in Unix (below)
+                    throw new AssumptionViolatedException("Copying files in Windows is only supported if a LocalController is in use. Test will be skipped.");
```

Comment:
The scope of #84 is to make tests work on Windows.

The scope of my PR is to implement a test suite that _has_ to be executed on a Windows machine (to test MSBuild). 

If I remove the LocalController check, and a non LocalController is used and tests are being run on Windows, this piece of code will not do what is expected and we would start getting non informative error messages. So, I am not sure what you are suggesting to do here. Can you explain?